packages/amplify-e2e-tests/src/__tests__/rds-refers-to-fields.test.ts

```diff
@@ -0,0 +1,806 @@
+import {
+  RDSTestDataProvider,
+  addApiWithoutSchema,
+  addRDSPortInboundRule,
+  amplifyPush,
+  createNewProjectDir,
+  createRDSInstance,
+  deleteDBInstance,
+  deleteProject,
+  deleteProjectDir,
+  getAppSyncApi,
+  getProjectMeta,
+  importRDSDatabase,
+  initJSProjectWithProfile,
+  removeRDSPortInboundRule,
+} from 'amplify-category-api-e2e-core';
+import { existsSync, writeFileSync } from 'fs-extra';
+import generator from 'generate-password';
+import path from 'path';
+import AWSAppSyncClient, { AUTH_TYPE } from 'aws-appsync';
+import { GQLQueryHelper, runQuery } from '../query-utils/gql-helper';
+
+// to deal with bug in cognito-identity-js
+(global as any).fetch = require('node-fetch');
+
+describe('RDS refersTo directive on model fields', () => {
+  const publicIpCidr = '0.0.0.0/0';
+  const [db_user, db_password, db_identifier] = generator.generateMultiple(3);
+
+  // Generate settings for RDS instance
+  const username = db_user;
+  const password = db_password;
+  const region = 'us-east-1';
+  let port = 3306;
+  const database = 'default_db';
+  let host = 'localhost';
+  const identifier = `integtest${db_identifier}`;
+  const projName = 'rdsreferstoapitest2';
+
+  let projRoot;
+  let appSyncClient;
+
+  beforeAll(async () => {
+    projRoot = await createNewProjectDir('rdsreferstoapi2');
+    await setupDatabase();
```

Comment:
Modify this step to use the VPC configuration and reduce the amount of time the ports are kept open. Refer to the below PR for more details.
https://github.com/aws-amplify/amplify-category-api/pull/1875/files